Procfile

```diff
@@ -1,13 +1,46 @@
+###############################
+#         DEVELOPMENT         #
+###############################
+
 # Procfile for development using the new threaded worker (scheduler, twitter stream and delayed job)
 web: bundle exec rails server -b0.0.0.0
 jobs: bundle exec rails runner bin/threaded.rb
 
-# Possible Profile configuration for production:
-# web: bundle exec unicorn -c config/unicorn/production.rb
+# Old version with separate processes (use this if you have issues with the threaded version)
+# web: bundle exec rails server
+# schedule: bundle exec rails runner bin/schedule.rb
+# twitter: bundle exec rails runner bin/twitter_stream.rb
+# dj: bundle exec script/delayed_job run
+
+###############################
+#         PRODUCTION          #
+###############################
+
+# Using the threaded worker (consumes less RAM but can run slower)
+# web: bundle exec unicorn -c config/unicorn.rb
```

Comment:
Maybe a comment saying that they need to create `config/unicorn.rb` and link it in?